src/cluster.py

```diff
@@ -492,8 +556,10 @@ def render_patroni_yml_file(
             version=self.get_postgresql_version().split(".")[0],
             minority_count=self.planned_units // 2,
             pg_parameters=parameters,
+            primary_cluster_endpoint=self.charm.async_replication.get_primary_cluster_endpoint(),
+            extra_replication_endpoints=self.charm.async_replication.get_standby_endpoints(),
         )
-        self.render_file(f"{PATRONI_CONF_PATH}/patroni.yaml", rendered, 0o600)
+        self.render_file(f"{PATRONI_CONF_PATH}/patroni.yaml", rendered, 0o644)
```

Comment:
Q: who else should be able to read patroni.yaml? 
I mean, I can imagine group read requirements, but you are exposing read to 'other' as well.

Reply:
There is no need for others to read. I reverted this change on https://github.com/canonical/postgresql-operator/pull/452/commits/bbd5976285a1d0072cba4f6854d531c85b3c2594.